lib/shelley/src/Cardano/Wallet/Shelley/Compatibility.hs

```diff
@@ -502,6 +502,8 @@ fromShelleyPParams bound pp = W.ProtocolParameters
         desiredNumberOfStakePoolsFromPParams pp
     , minimumUTxOvalue =
         minimumUTxOvalueFromPParams pp
+    , poolDeposit = poolDepositFromPParams pp
+    , keyDeposit = keyDepositFromPParams pp
```

Comment:
stakeKeyDeposit would be a slightly better name, there are many things that can qualify as "key" in the application.